src/ru/nsu/ccfit/zuev/osu/scoring/ScoringScene.java

```diff
@@ -82,6 +83,47 @@ public void load(final StatisticV2 stat, final TrackInfo track,
         String mapperStr = "Beatmap by " + trackInfo.getCreator();
         String playerStr = "Played by " + stat.getPlayerName() + " on " +
                 new java.text.SimpleDateFormat("yyyy/MM/dd HH:mm:ss", Locale.getDefault()).format(new java.util.Date(stat.getTime()));
+        if (stat.getChangeSpeed() != 1 || stat.isEnableForceAR()){
+            playerStr += " [";
+            if (stat.getChangeSpeed() != 1){
+                playerStr += String.format("%.2fx,", stat.getChangeSpeed());
+            }
+            if (stat.isEnableForceAR()){
+                playerStr += String.format("AR%.1f,", stat.getForceAR());
+            }
+            if (playerStr.endsWith(",")){
+                playerStr = playerStr.substring(0, playerStr.length() - 1);
+            }
+            playerStr += "]";
+        }
+        //calculatePP
+        if (Config.isDisplayScorePP()){
+            StringBuilder ppinfo = new StringBuilder();
+            ppinfo.append("[");
+            DifficultyReCalculator diffReCalulator = new DifficultyReCalculator();
+            float newstar = diffReCalulator.reCalculateStar(
+                            trackInfo,
+                            stat.getSpeed(), 
+                            diffReCalulator.getCS(stat, trackInfo));
+            diffReCalulator.calculaterPP(stat, trackInfo);
+            double pp = diffReCalulator.getTotalPP();
+            double aimpp = diffReCalulator.getAimPP();
+            double spdpp = diffReCalulator.getSpdPP();
+            double accpp = diffReCalulator.getAccPP();
+            diffReCalulator.calculaterMaxPP(stat, trackInfo);
+            double max_pp = diffReCalulator.getTotalPP();
+            double max_aimpp = diffReCalulator.getAimPP();
+            double max_spdpp = diffReCalulator.getSpdPP();
+            double max_accpp = diffReCalulator.getAccPP();
+            ppinfo.append(String.format("%.2f*,", newstar));
+            ppinfo.append(String.format("PP:%.2f/%.2f(", pp, max_pp));
```

Comment:
I think pp display should only show pp value (no max pp, aim pp, speed pp, and acc pp) since all these information are cluttering result screen and seemingly unnecessary for a user-friendly environment.

Reply:
pp display is an advanced option(off by default), so I think display more pp info is better. 
And It's just an experimental features. If it's a complete pp system, I think only show pp value will be better.

Reply:
Alright then.